test/autobot.test.ts

```diff
@@ -0,0 +1,52 @@
+import { Autobot, PRContext } from "../src/autobot";
+import { Application } from "probot";
+
+const app = (Symbol("app") as unknown) as Application;
+const context = ({} as unknown) as PRContext;
+
+describe("autobot", () => {
+  describe("start", () => {
+    it("creates a new autobot instance", () => {
+      const autobot = Autobot.start(app, []);
+      expect(autobot).toBeInstanceOf(Autobot);
+    });
+
+    it("calls the onStart hook with the correct arguments", () => {
+      const autobot = Autobot.start(app, []);
+      // @ts-ignore
+      expect(autobot.hooks.app.onStart.call).toBeCalledWith(app);
```

Comment:
The nice thing about being able to access private variables (when you silence typescript) is that it makes it easier to test.